chirp/drivers/tdh8.py

```diff
@@ -2144,6 +2091,15 @@ class TDH8_GMRS(TDH8):
     _vhf_range = (136000000, 175000000)
     _uhf_range = (400000000, 521000000)
 
+    def validate_memory(self, mem):
+        msgs = super().validate_memory(mem)
+        if mem.freq / 1000000 in GMRS_FREQS3:
+            pass
+        else:
+            msgs = ("The frequency in channels 31-55 must be between "
+                    "462.55000-462.72500 in 0.025 increments.")
```

Comment:
`msgs` needs to be a list of objects, not a single string. This will break the UI. You also need to append, in case the base `validate_memory()` returned any messages.

You're requiring all memories to be in that range now, because you're not checking `mem.number`. You need to do something like this:
```
if 31 <= mem.number <= 55: and mem.freq not in GMRS_FREQS:
        msgs.append(chirp_common.ValidationError("The frequency in channels ...")
```
...right?